src/Avalonia.Visuals/Media/Fonts/FontFamilyKey.cs

```diff
@@ -12,48 +11,26 @@ namespace Avalonia.Media.Fonts
     public class FontFamilyKey
```

Comment:
Does it have to be a class? There is a lot of boilerplate for comparison and GetHashCode. Another approach would be to use [Equals.Fody](https://github.com/Fody/Equals)

Reply:
Just felt cleaner when I introduced this.  I am fine with refactoring. The current solution isn't too much code I think.